sdcm/nemesis.py

```diff
@@ -1701,6 +1699,25 @@ def disrupt_mgmt_repair_cli(self):
             mgr_task.id, str(mgr_task.status))
         self.log.info('Task: {} is done.'.format(mgr_task.id))
 
+    @latency_calculator_decorator
+    def disrupt_mgmt_repair_kubernetes(self):
```

Comment:
I'm not sure why we need duplicate of this nemesis ? how it's different from the regular one ?

Reply:
is this temporary ?

Reply:
It is totally different nemesis, it tests operator feature.
Eventhou this feature uses scylla-manager under the hood, the target here is scylla-operator, not manager.

Reply:
I understand it, but the flow of the nemesis should be identical, now if we have thing we need to add to it, or fix in this flow.
we have two places.

we are using this nemesis also on cloud, as you've seen, without adding a new nemesis specifically for it.

anyhow that's for @bentsi to decide. 
I find it weird we duplicate those things, if there a current bug we can put and if in the right place based on the `cluster_backend`.

Reply:
@fruch agree, we need to have a new flavor of manager cluster for Operator that will have the same API so we won't duplicate the nemesis code. Dmitry, please improve.

Reply:
redone